README.md

```diff
@@ -13,27 +13,81 @@ Zubhub is designed and developed by [Unstructured Studio](https://unstructured.s
-- Run `make init` to do all the initial setups required and start the server.
-  `make init` generates the minimal .env file required to run the backend, 
-  spins-up all containers defined in the backend docker-compose file,
-  applies all necessary migrations to the database,
-  and creates a default admin user with 
-  username and password of dummy and dummy_password respectively.
+1. install [docker](https://docs.docker.com/get-docker/).
+2. install [docker-compose](https://docs.docker.com/compose/install/).
+3. Install [Node.js 14 and npm 7 or later](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm/).
+4. Install [make](https://www.gnu.org/software/make/) using either your system’s package manager (Linux) or Xcode command line developer tools (OSX). On Windows, you can use [MozillaBuild](https://wiki.mozilla.org/MozillaBuild).
 
-  Subsequently to start and stop the docker containers, use `make start`, `make stop`, or `make down`. (run `make help` to see all the available make commands).
-- Visit `localhost:8000` on your browser to access the API documentation.
+### Quick Start
+The following describes how to set up an instance of the site on your computer for development with Docker and docker-compose.
 
-### Frontend
+1. Clone the [zubHub repository](https://github.com/unstructuredstudio/zubhub)
+ ```sh
+       $ git clone https://github.com/mozilla/pontoon.git
+ ```
+> **_NOTE:_** ❗  : To contribute changes to the project, you will need to [fork](<https://help.github.com/en/github/getting-started-with-github/fork-a-repo>) the repository under your own GitHub account.
+
+#### Backend
+2. From the root of the repository, run:
+```sh
+       $ cd ./zubhub_backend
+ ```
+3. Run:
+
+```sh
+       $ make init
+```
+This will run all  the initial setups required and start the server.generates the minimal `.env file` required to run the backend, spins-up all containers defined in the backend docker-compose file,applies all necessary migrations to the database,and creates a default admin user with username and password of `dummy` and `dummy_password` respectively.
+
+Incase you're facing issues after running the above, like make: Error,
+kindly run this before the above command:
+
+```sh
+       $ docker-compose up
+```
+
+4. Run the server:
+
+Subsequently to start and stop the docker containers, you run the following
+
+To start: 
+```sh
+       $ make start
+```
+To stop: 
+```sh
+       $ make stop or make down
+```
+For other make commands: run `make help`
+
+Visit  http://localhost:8000 on your browser to access the API documentation.
+
+#### Frontend
 - Create a file named .env in the frontend root folder (same directory with package.json)
 - Copy the content of .env.example and paste into the new .env file.
 - Run `make start` to spin up the frontend container.
 - Visit `localhost:3000` on your browser to access the frontend.
 
+### Installing Docker on Windows Pro/Enterprise/Education
```

Comment:
Since this is a Windows specific section and not mandatory for everyone to do this, we need to specify this by putting this under Troubleshooting or Windows tools setup or some other section

Reply:
> Since this is a Windows specific section and not mandatory for everyone to do this, we need to specify this by putting this under Troubleshooting or Windows tools setup or some other section

It's not mandatory, that's why i wrote the title as "Installing Docker on Windows", so anyone with a windows OS can easily use that or else ignore, or what do you suggest, which section do you think fit well to put it, you can specify, i'll do that.

Reply:
@NdibeRaymond i don't really understand, what title do you suggest, let me just change it accordingly